### ISSUE TITLE
a11y: Identical page title

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13849]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Identical page title is used for mentioned pages which made it difficult for screen reader users to identify the type of information available on each page.

Page title "Rancher" is provided for "Login" and "Homepage" pages.

Provide meaningful and unique title for each webpage within the <title> tag in page source.
In present instance specify unique page title as "Login - Rancher" and "Homepage - Rancher".

**To Reproduce**

**Result**

**Expected Result**

The screen reader should announce a meaningful and descriptive page title that gives information about content of webpage for their users.

**Screenshots**

**Additional context**

**Comment by yonasberhe23** [comment]
e2e test adequately covers this update. No manual validation is required. Closing this ticket
